ansi/sixel/decoder.go

```diff
@@ -288,6 +288,11 @@ func ParseRaster(data io.Reader) (pan, pad, ph, pv int, err error) {
 	return
 }
 
+func ParseRepeat(data io.Reader) (count int, r byte, err error) {
+	_, err = fmt.Fscanf(data, "%d%b", &count, &r)

```

Comment:
I would take a `[]byte` over a reader for simplicity and so that it works with raw bytes